### DISCUSSION TITLE
pcl2什么时候恢复联机功能？

### DISCUSSION THREAD

**Discussion by Alpha9891** [Meloong-Git/PCL#3366] [提问] [answered]
龙猫赶紧恢复联机啊啊啊啊啊啊啊啊啊啊啊啊

**Comment by MoYuan-CN** [comment, marked as answer]
*“至于联机，因为网易还在继续代理中国版，我和 HMCL 的维护者都觉得还得谨慎一点……况且现在也有 Mod 支持联机了……咳咳。” ——龙腾猫跃*

BakaXL 现已于 2023 年 10 月 8 日将其部分服务与 Octo 章鱼网络进行对接，并对其可用性进行初步评估，待评估完成后，Octo 网络会全面对其他启动器开放

请注意，Octo 开放不等价于 PCL 上线联机，具体信息以龙猫发布的消息为准，仅供参考。届时切勿散布任何未经相关启动器作者及服务提供方证实的信息。

---

参考：#1930 [2.6.11 更新日志](https://www.bilibili.com/read/cv28121157/)

**Comment by allMagicNB** [comment]
# 你急啥呢急急国王，这么急用BakaXL去

**Reply by Ad-closeNN** [reply]
~~我知道你很急，但是你先别急~~

**Comment by Ad-closeNN** [comment]
# 要么用

**1.简幻欢（50积分/天即可，一天签到50积分，需要交1元认证费）**
**2.联机Mod**
**3.叫那运营商给你开IPV6**
**4.联机工具**
~~5.来你家玩MC~~

**Reply by Deep-Dark-Forest** [reply]
我还未成年.jpeg

**Reply by Ad-closeNN** [reply]
~~让别人来你家玩MC~~
~~IPV6也是个好选择~~

**Comment by WForst-Breeze** [comment]
别急
